.orchestra/config/components/toolchain/lib/binutils.lib.yml

```diff
@@ -51,9 +53,14 @@ configure: |
     (@= expand_args(extra_binutils_configure_options) @) \
     CFLAGS="-w -ggdb3 -O3" \
     CXXFLAGS="-w -ggdb3 -O3"
+post_install: |
+  (@- if "win" in toolchain_name: @)
+  # Symlink needed for making windres use the right gcc
+  ln -s "../../gcc-bin/(@= gcc_version @)/(@= triple @)-gcc" "${TMP_ROOT}(@= bindir @)/(@= triple @)-gcc"
+  (@ end -@)
```

Comment:
Mmmmh, this is highly suspicious.
You're creating a symlink to a file that does not exist yet.
If you really want to do this, do it from the GCC component.
One day we should merge `gcc-bin` and `binutils-bin`, which is an artifact from stuff copied from Gentoo.

Also, do we really need to do `if "win" in toolchain_name`? It's ugly and I don't think an extra symlink will make anyone unhappy.

Also, are you sure this issues is not solved by adding to the GCC component `add_to_path: .../gcc-bin/...`?

Reply:
Ping

Reply:
`add_to_path` doesn't work because it appears `windres` uses a hardcoded list of paths to try when searching for binaries, which was broken due to the `{gcc,binutils}-bin` split. I'll move the symlink creation in gcc.